docs/tutorials/getting_started/windows.rst

```diff
@@ -0,0 +1,56 @@
+Windows Installation Guide
+==========================
+
+Quickstart
+----------
+The following commands should be sufficient to get you started with installing and using IDAES. For more information on these steps, see below.
+
+.. sidebar:: Learn more about the Installation
+   :class: quickstart-toc
+
+   * `Install Prerequisites`_
+   * `Install IDAES-PSE`_
+   * `Install IDAES Extensions`_
+   * `Install IDAES Examples`_
+   * `Run IDAES Tests`_
+
+.. Import quick start guide, including OS specific lines & skipping non-OS lines & comments
+.. literalinclude:: install_templates/quickstart.md
+   :language: bash
+   :linenos:
+   :lines: 1-5,7,10-
```

Comment:
These might need to change if the file `quickstart.md` is modified, right?

Perhaps then add a comment in `quickstart.md` that it is referenced elsewhere by line numbers so would need to be checked if that file is changed.

Reply:
Really wanted to find something slicker than the direct line references, but I ended up going with adding a comment warning for developers.